etc/examples/caffe-gpu-model/manifest.yml

```diff
@@ -0,0 +1,25 @@
+name: mnist-caffe-gpu-model
```

Comment:
Same as above - can`t these be instructions like change these params in manifest + proto files, and then run? This way users also learn